sklearn/linear_model/tests/test_passive_aggressive.py

```diff
@@ -1,3 +1,5 @@
+import unittest
+from nose.tools import assert_true
```

Comment:
Please use instead `from sklearn.utils.testing import assert_true` which centralizes the nose dependency.